scripts/Project/ProjectPython.py

```diff
@@ -69,24 +90,11 @@ def transform(code: str, module: str) -> str:
                 continue
             with open(fuzzer_path, "r") as f:
                 code = f.read()
-            target_module = self.get_target_module(code)
-            if target_module is None:
-                warning(f"Couldn't find target module for {fuzzer_path}")
-                continue
-
             with open(f"{self.project_oss_dir}/decorated_{fuzzer}", "w") as f:
-                f.write(transform(code, target_module))
+                f.write(transform(code))
 
         self.build_w_pass()
 
-        # Remove decorated_{fuzzer} files
-        for fuzzer in self.fuzzers:
-            fuzzer_path = f"{self.project_oss_dir}/{fuzzer}"
-            if not path.isfile(fuzzer_path):
-                warning(f"{fuzzer_path} not found")
-                continue
-            os.system(f"rm {self.project_oss_dir}/decorated_{fuzzer}")
-
     def get_target_module(self, code: str) -> Optional[str]:
```

Comment:
I think we can safely remove `get_target_module` with the new `transform` method